clients/admin-ui/src/features/common/tablev2/TableSkeletonLoader.tsx

```diff
@@ -0,0 +1,16 @@
+import { Skeleton, Stack } from "@fidesui/react";
+import { FC } from "react";
+
+type Props = {
+  rowHeight: number;
+  numRows: number;
+};
+
+export const TableSkeletonLoader: FC<Props> = ({ rowHeight, numRows }) => {
+  const rows = [];
+  for (let i = 0; i < numRows; i += 1) {
+    rows.push(<Skeleton height={`${rowHeight}px`} key={i} />);
+  }
+
+  return <Stack>{rows}</Stack>;
```

Comment:
this feels more react-y to me, and then I don't think you have to do the for loop plus rows array:

```js
<Stack>
  {[...Array(numRows).keys()].map(key => {
  <Skeleton height=`${rowHeight}px`} key={key} />
})}
</Stack>
```

Reply:
I actually tried that same method. It wasn't working when I tried and had no errors. I fell back to other method that's currently in use.